nvflare/private/fed/server/server_app_runner.py

```diff
@@ -74,3 +79,6 @@ def sync_up_parents_process(self, args, server):
 
     def update_job_run_status(self, server):
         server.engine.update_job_run_status()
+
+    def stop(self):
+        self.server.engine.asked_to_stop = True
```

Comment:
I prefer have some kind of method introduced instead of directly access other class's variable.

Reply:
just a simple line, maybe a Python way.